app/controllers/public_health_controller.rb

```diff
@@ -7,100 +7,111 @@ class PublicHealthController < ApplicationController
   def exposure
     # Restrict access to public health only
     redirect_to(root_url) && return unless current_user.can_view_public_health_dashboard?
-
-    @all_count = current_user.viewable_patients.where(isolation: false).where(purged: false).size
-    @i_all_count = current_user.viewable_patients.where(isolation: true).where(purged: false).size
-    @symptomatic_count = current_user.viewable_patients.exposure_symptomatic.size
-    @non_reporting_count = current_user.viewable_patients.exposure_non_reporting.size
-    @asymptomatic_count = current_user.viewable_patients.exposure_asymptomatic.size
-    @pui_count = current_user.viewable_patients.exposure_under_investigation.size
-    @closed_count = current_user.viewable_patients.monitoring_closed_without_purged.where(isolation: false).size
-    @transferred_in_count = current_user.jurisdiction.transferred_in_patients.where(isolation: false).size
-    @transferred_out_count = current_user.jurisdiction.transferred_out_patients.where(isolation: false).size
-    @assigned_jurisdictions = Hash[Jurisdiction.order(:path).find(current_user.jurisdiction.subtree_ids).pluck(:id, :path).map { |id, path| [id, path] }]
-    @assigned_users = current_user.jurisdiction.all_assigned_users
   end
 
   def isolation
     # Restrict access to public health only
     redirect_to(root_url) && return unless current_user.can_view_public_health_dashboard?
-
-    @all_count = current_user.viewable_patients.where(isolation: true).where(purged: false).size
-    @e_all_count = current_user.viewable_patients.where(isolation: false).where(purged: false).size
-    @requiring_review_count = current_user.viewable_patients.isolation_requiring_review.size
-    @non_reporting_count = current_user.viewable_patients.isolation_non_reporting.size
-    @reporting_count = current_user.viewable_patients.isolation_reporting.size
-    @closed_count = current_user.viewable_patients.monitoring_closed_without_purged.where(isolation: true).size
-    @transferred_in_count = current_user.jurisdiction.transferred_in_patients.where(isolation: true).size
-    @transferred_out_count = current_user.jurisdiction.transferred_out_patients.where(isolation: true).size
-    @assigned_jurisdictions = Hash[Jurisdiction.order(:path).find(current_user.jurisdiction.subtree_ids).pluck(:id, :path).map { |id, path| [id, path] }]
-    @assigned_users = current_user.jurisdiction.all_assigned_users
   end
 
   def patients
     # Restrict access to public health only
     redirect_to(root_url) && return unless current_user.can_view_public_health_dashboard?
 
     # Validate workflow param
```

Comment:
these validate code blocks are good candidates for `before_action`s

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/commit/7f06c26894cd5271163119a8b04b909943f4bd46#diff-f332c4e8d749e2f0c02c18036420ef37R277